src/Microsoft.AspNetCore.Razor.Evolution/RazorEngine.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Evolution.TagHelpers;
```

Comment:
Please hold off on adding new namespaces for now. I want to see the shape of things first.